lib/ansible/modules/cloud/ovirt/ovirt_host_storage_facts.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_host_storage_facts
+short_description: Retrieve facts about one or more oVirt/RHV HostStorages
+author: "Daniel Erez (@derez)"
+version_added: "2.4"
+description:
+    - "Retrieve facts about one or more oVirt/RHV HostStorages."
+options:
+    host:
+        description:
+            - "Host to get device list from."
+        required: true
+    iscsi:
+        description:
+            - "Dictionary with values for iSCSI storage type:"
+            - "C(address) - Address of the iSCSI storage server."
+            - "C(target) - The target IQN for the storage device."
+            - "C(username) - A CHAP user name for logging into a target."
+            - "C(password) - A CHAP password for logging into a target."
+            - "Note that these parameters are not idempotent."
```

Comment:
please remove `Note that these parameters are not idempotent.`
It facts module, so we don't care about that.